receiver/jaegerreceiver/trace_receiver.go

```diff
@@ -278,6 +289,30 @@ func (jr *jReceiver) SubmitBatches(ctx thrift.Context, batches []*jaeger.Batch)
 	return jbsr, nil
 }
 
+func (jtr *jTchannelReceiver) SubmitBatches(ctx thrift.Context, batches []*jaeger.Batch) ([]*jaeger.BatchSubmitResponse, error) {
```

Comment:
Can this and the other SubmitBatches function may be share call a shared internal func instead of duplicating?

Reply:
Just to confirm: before that same signature satisfied two interfaces (`JaegerBatchesHandler`and `TChanCollector`) in version 1.14 those diverged, correct?

Reply:
Yes, precisely @pjanotti